src/main/java/tds/exam/results/configuration/ExamResultsTransmitterServiceProperties.java

```diff
@@ -68,6 +69,19 @@ public void setTisUrl(final String tisUrl) {
         this.tisUrl = tisUrl;
     }
 
+    /**
+     * Gets the callback url for the Test Integration System
+     *
+     * @return the TIS callback URL
+     */
+    public String getTisCallbackUrl() {
```

Comment:
How is this used?  And what value do we expect to put here?  Or is this code we've added and we'll need to implement a endpoint to handle the callback?